### ISSUE TITLE
new version of obs uses flashback

### ISSUE THREAD

**Issue by flover-luffy** [dd-center/obs-streamlink#38]
Version 29.12 is not supported

**Comment by cqjjjzr** [comment]
Indeed. Not having time recently to fix this, will look into this later.

Maybe it's really the time to switch from async video/audio playback to synchornized one, since such part caused most of the issues (audio not being played, frequent buffering etc.)

**Comment by flover-luffy** [comment]
Thank you for your hard work
wait for the problem to be solved

**Comment by caveman99** [comment]
Same with Version 30. Problem persists. (no pressure, just a ping)
